source/MulensModel/binarylenswithshear.py

```diff
@@ -0,0 +1,617 @@
+             zeta_conj, z1 * K_pow3 * zeta_conj, -3 * z1 * K_pow2 *
+             zeta_conj, 3 * z1 * convergence_K * zeta_conj, -z1 *
+             zeta_conj, -3 * shear_G * Gc * zeta_conj_pow2, K_pow2
+             * zeta_conj_pow2, -2 * convergence_K * zeta_conj_pow2,
+             zeta_conj_pow2])
+        coeff_6 = c_sum(
+            [4 * total_m * zeta * Gc * convergence_K, -4 * total_m * zeta * Gc,
+             15 * total_m * z1 * shear_G * Gc_pow2, -4 * total_m * z1 * shear_G
+             * Gc * convergence_K, 4 * total_m * z1 * shear_G * Gc, -4 *
+             total_m * z1 * Gc * K_pow2, 8 * total_m * z1 * Gc * convergence_K,
+             -4 * total_m * z1 * Gc, total_m * z1 * K_pow3, -3 * total_m * z1 *
+             K_pow2, 3 * total_m * z1 * convergence_K, -total_m * z1, -12 *
+             total_m * shear_G * Gc * zeta_conj, 2 * total_m * K_pow2 *
+             zeta_conj, -4 * total_m * convergence_K * zeta_conj, 2 * total_m *
+             zeta_conj, 3 * zeta * z1_pow2 * Gc_pow2 * convergence_K, -3 * zeta
+             * z1_pow2 * Gc_pow2, -3 * zeta * z1_pow2 * Gc * K_pow2, 6 * zeta *
+             z1_pow2 * Gc * convergence_K, -3 * zeta * z1_pow2 * Gc, -6 * zeta
+             * z1 * Gc * convergence_K * zeta_conj, 6 * zeta * z1 * Gc *
+             zeta_conj, zeta * z1 * K_pow2 * zeta_conj, -2 * zeta * z1 *
+             convergence_K * zeta_conj, zeta * z1 * zeta_conj, zeta *
+             convergence_K * zeta_conj_pow2, -zeta * zeta_conj_pow2, z1_pow3 *
+             shear_G * Gc_pow3, -3 * z1_pow3 * shear_G * Gc_pow2 *
+             convergence_K, 3 * z1_pow3 * shear_G * Gc_pow2, -z1_pow3 * Gc_pow2
+             * K_pow2, 2 * z1_pow3 * Gc_pow2 * convergence_K, -z1_pow3 *
+             Gc_pow2, 3 * z1_pow3 * Gc * K_pow3, -9 * z1_pow3 * Gc * K_pow2, 9
+             * z1_pow3 * Gc * convergence_K, -3 * z1_pow3 * Gc, -9 * z1_pow2 *
+             shear_G * Gc_pow2 * zeta_conj, 6 * z1_pow2 * shear_G * Gc *
+             convergence_K * zeta_conj, -6 * z1_pow2 * shear_G * Gc *
+             zeta_conj, 6 * z1_pow2 * Gc * K_pow2 * zeta_conj, -12 * z1_pow2 *
+             Gc * convergence_K * zeta_conj, 6 * z1_pow2 * Gc * zeta_conj, -3 *
+             z1_pow2 * K_pow3 * zeta_conj, 9 * z1_pow2 * K_pow2 * zeta_conj, -
+             9 * z1_pow2 * convergence_K * zeta_conj, 3 * z1_pow2 * zeta_conj,
+             3 * z1 * shear_G * Gc_pow2 * m_diff, 9 * z1 * shear_G * Gc *
+             zeta_conj_pow2, -z1 * shear_G * convergence_K * zeta_conj_pow2, z1
+             * shear_G * zeta_conj_pow2, -2 * z1 * Gc * K_pow2 * m_diff, 4 * z1
+             * Gc * convergence_K * m_diff, -2 * z1 * Gc * m_diff, -z1 * K_pow3
+             * m_diff, 3 * z1 * K_pow2 * m_diff, -3 * z1 * K_pow2 *
+             zeta_conj_pow2, -3 * z1 * convergence_K * m_diff, 6 * z1 *
+             convergence_K * zeta_conj_pow2, z1 * m_diff, -3 * z1 *
+             zeta_conj_pow2, -shear_G * zeta_conj_pow3])
+        coeff_5 = c_sum(
```

Comment:
It may not be possible to fix this, but it makes me nervous that a "single line" of code takes up an entire page. This makes it difficult to read and maintain. One possible improvement would be to start a new line after every comma to make the individual terms more visually distinct.

Reply:
I makes me a bit nervous as well, but I don't see how to make it much better.